parsec/backend/postgresql/organization.py

```diff
@@ -99,14 +99,32 @@
 """
 )
 
+_q_check_users_limit = Q(
+    f"""
+    SELECT
+        organization._id
+    FROM organization
+    LEFT JOIN
+        user_ on user_.organization = organization._id
+    WHERE
+        organization = { q_organization_internal_id("$organization_id") }
```

Comment:
`organization.organization_id = $organization_id` should do the trick here (that's basically what `q_organization_internal_id` does, but with an aditional `SELECT` we don't need here given we already have our hands on the organization table)